src/OpenConext/Profile/Value/Consent.php

```diff
@@ -95,4 +113,12 @@ public function getConsentType()
     {
         return $this->consentType;
     }
+
+    /**
+     * @return null|AttributeSet
+     */
+    public function getAttributes()
+    {
+        return $this->attributes;
```

Comment:
Released attributes sounds fine to me, as `releasing` is what is consented to. Has-er + LogicException seems the way to go with a getter like this.